code/src/App.js

```diff
@@ -1,9 +1,92 @@
-import React from 'react';
+/* eslint-disable */
+
+import React, { useState } from 'react';
+import { ThoughtsList } from 'components/ThoughtsList';
+import { useEffect } from 'react';
+import { NewThoughtsForm } from 'components/NewThoughtsForm';
 
 export const App = () => {
+  const [thoughtsList, setThoughtsList] = useState([]);
+  const [loading, setLoading] = useState(true);
+  const [newThoughts, setNewThoughts] = useState("");
+  const [newLike, setNewLike] = useState(false);
+  const [count, setCount] = useState(0);
+
+  // create function to fetch thoughts from database
```

Comment:
Easy to follow your code! Just a little side note to clean up the notes you've kept in the code too ;)